<?xml version="1.0"?>
<launch>
    <!-- vision lidar fusion launch script -->
    <param name="euclidean_cluster_downsampling_size" type="double" value="0.1"/>
    <param name="min_cluster_size" type="int" value="25"/>
    <param name="max_cluster_size" type="int" value="200000"/>
    <param name="cluster_tolerance" type="double" value="0.25"/>
    <node name="vision_lidar_fusion_node" pkg="vision_lidar_fusion" type="vision_lidar_fusion_node" output="screen"><!-- launch-prefix="xterm -g 200x60 -e gdb -ex run \-\-args"-->
        <param name="visualize" type="bool" value="true"/>
        <param name="config_file_path" type="string" value="/home/gi/GAAS/config/camera1_lidar1.yaml"/>
    </node>
    <node name="euclidean_cluster_extration_node" pkg="vision_lidar_fusion" type="euclidean_cluster_extraction" output="screen"/> <!--launch-prefix="stterm -g 200x60 -e gdb -ex run \-\-args"-->
</launch>

